doc/workflows.mld

```diff
@@ -0,0 +1,50 @@
+{0 OCamlFormat Manual}
+
+{1 Workflows}
```

Comment:
I'd rename this to How-Tos and add a small description of the description. It wasn't obvious what the goal was until I read a couple of sub-sections.

Reply:
done